tests/test_data/test_dataset.py

```diff
@@ -26,6 +27,40 @@ def test_classes():
         get_classes('unsupported')
 
 
+def test_classes_file_path():
+    train_pipeline = [dict(type='LoadImageFromFile')]
+    classes_path = '../data/classes.txt'
+
+    # classes.txt with full categories
+    categories = get_classes('cityscapes')
+    with open(classes_path, 'w') as f:
+        f.write('\n'.join(categories))
+
+    kwargs = dict(
+        pipeline=train_pipeline,
+        img_dir='./',
+        classes=classes_path
+    )
```

Comment:
![image](https://user-images.githubusercontent.com/16716329/135150567-1b6fa030-b0ce-47b5-972c-123ee2966968.png)

use yapf to format the code (pre-commit hook)

Reply:
https://github.com/open-mmlab/mmsegmentation/pull/918/checks?check_run_id=3735897729